app/views/posts/_new_request.html.erb

```diff
@@ -1,16 +1,9 @@
-<div class="container">
```

Comment:
same here. we do want to keep every page inside of a <div class="container mt-3"> wrapper

Reply:
ah! nevermind, i see you render these are partials. good stuff!